components/email/src/init.spec.js

```diff
@@ -740,3 +1232,210 @@ describe("Email: Toggle email of sender/recipient", () => {
     });
   });
 });
+
+describe("Should Render Reply Button And Dispatch Event When Clicked", () => {
+  beforeEach(() => {
+    cy.visit("/components/email/src/cypress.html");
+
+    cy.get("nylas-email").first().as("email");
+
+    cy.get("@email").invoke("attr", "show_reply", "true");
```

Comment:
You could alias the reply button here (like you did on L1322) rather than writing `cy.get("@email").find("div.reply button")` in all the tests
```suggestion
    cy.get("@email").invoke("attr", "show_reply", "true");
    cy.get("div.reply button").as("replyButton"); // or
    cy.get("@email").find("div.reply button").as("replyButton");
```